meta-mender-qemu/Dockerfile

```diff
@@ -11,6 +11,8 @@
 #       Use BUILDDIR from a poky build as image input.
 # -v <config-dir>:/mnt/config:ro
 #       Use server.crt from config-dir, if it exists.
+# -v <file>:/mnt/config/artifact-verify-key.pem:ro
```

Comment:
Can you mount a single file? I have to admit that I haven't tried, I was assuming the folder would be mounted and the file would just be in there. In that case it would share mount point with `server.crt`, but there should be a comment about the verify key, of course.